client/src/App.vue

```diff
@@ -194,12 +197,11 @@ export default class App extends Mixins(SessionTimerMixin) {
   async onPaymentModalActivated () {
     const { nrId } = newRequestModule
     const componentName = newRequestModule.displayedComponent
+
     // Only do this for New NRs!!!
     if (nrId && ['SubmissionTabs'].indexOf(componentName) > -1) {
-      if (this.$NR_COMPLETION_TIMEOUT_MS > 0) {
```

Comment:
Don't need to check this since "stopTimer" can be called when the timer doesn't exist.